src/mod_private_mysql.erl

```diff
@@ -1,62 +0,0 @@
-%%% NS is namespace or key.
-%%% XML is #xmlel{} or value.
--module(mod_private_mysql).
--author('arcusfelis@gmail.com').
--behaviour(mod_private).
-
--export([init/2,
-         multi_set_data/3,
-         multi_get_data/3,
-         remove_user/2]).
-
--export([get_all_nss/2]).
-
--include("mongoose.hrl").
--include("jlib.hrl").
-
-init(_Host, _Opts) ->
-    ok.
-
-multi_set_data(LUser, LServer, NS2XML) ->
-    SLUser = mongoose_rdbms:escape_string(LUser),
-    Rows = [sql_row(NS, XML) || {NS, XML} <- NS2XML],
-    replace_like_insert_result(
-        rdbms_queries:multi_set_private_data(LServer, SLUser, Rows)).
```

Comment:
OK after reading up I understand that the new version should be performant because it is a query similar to REPLACE but it _updates_ instead of _delete+insert_,. This could be written in the commit message to justify removing this.

Reply:
The main reason to remove it is to maintain just one version.

There are several ways to optimize upserts though:
- add API to do multiple upserts
- use replaces for mysql, but behind the scene, no need for a separate module.